net2/config.json

```diff
@@ -142,6 +142,14 @@
       "userAgent": {
         "expires": 604800,
         "count": 10000
+      },
+      "sumflow": {
+        "expires": 10485760,
```

Comment:
value of 'expires' is used as parameter of zremrangebyscore. For sumflow and aggrflow, score indicates number of bytes transferred, not timestamp.

Reply:
what is the meaning for 10485760? i think for this case, just need to use "count" part, skip the expires part, maybe just expires to -1 is enough.